control-plane/subcommand/api-gateway/command_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package apigateway
```

Comment:
I feel like this test is probably completely unnecessary and will change as soon as the real command gets implemented. May remove it.

Reply:
I'm going to remove this, the bats tests are more useful at this point.